src/System.Data.Common/tests/System/Data/DataColumnExpressionTest.cs

```diff
@@ -202,7 +203,7 @@ public static IEnumerable<object[]> BinaryOperators()
                 foreach (var equation in comparisonEquations)
                 {
                     yield return new object[] { type, type, typeof(SqlBoolean), equation.Expression, operand1, operand2,  new SqlBoolean(equation.Result) };
-
+                    //TODO
```

Comment:
ditto

Reply:
It's fine to post questions in comments on the PR, but please don't add TODOs to the source. Thanks.